attributes/default.rb

```diff
@@ -4,6 +4,7 @@
 default['macos']['xcode']['version'] = '9.4.1'
 
 default['macos']['remote_login_enabled'] = true
+default['macos']['disable_vnc_screen_lock'] = false
```

Comment:
Double negative here - maybe `‘screen_lock_enabled’ = true`?

Reply:
Is the vnc screen lock separate and in a different plist than the default desktop screen lock? If so, would we not want to still include vnc? `‘vnc_screen_lock_enabled’ = true`